packages/web-components/src/components/va-combo-box/va-combo-box.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  forceUpdate,
+  Host,
+  Prop,
+  h,
+  State,
+  Fragment,
+} from '@stencil/core';
+import classnames from 'classnames';
+import { i18next } from '../..';
+import { comboBox } from './va-combo-box-library.js';
+
+/**
+ * @componentName Combo Box
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
+ * @guidanceHref form/combo-box
+ * @translations English
+ * @translations Spanish
+ */
+
+@Component({
+  tag: 'va-combo-box',
+  styleUrl: 'va-combo-box.scss',
+  shadow: true,
+})
+export class VaComboBox {
+  @Element() el: any;
+
+  @State() options: Array<Node>;
+
+  @State() labelNode: HTMLLabelElement;
+
+  /**
+   * Whether or not this is a required field.
+   */
+  @Prop() required?: boolean = false;
+
+  /**
+   * The combo box component will be disabled / read-only.
+   */
+  @Prop() disabled?: boolean = false;
+
+  /**
+   * Text label for the field.
+   */
+  @Prop() label!: string;
+
+  /**
+   * The placeholder string.
+   */
+  @Prop() placeholder?: string;
+
+  /**
+   * Name attribute for the select field.
+   */
+  @Prop() name!: string;
+
+  /**
+   * Selected value (will get updated on select).
+   */
+  @Prop({ reflect: true, mutable: true }) value?: string;
+
+  /**
+   * Error message to display. When defined, this indicates an error.
+   */
+  @Prop() error?: string;
+
+  /**
+   * Whether or not `aria-invalid` will be set on the inner select. Useful when
+   * composing the component into something larger, like a date component.
+   */
+  @Prop() invalid?: boolean = false;
+
+  /**
+   * Whether or not to fire the analytics events
+   */
+  @Prop() enableAnalytics?: boolean = false;
+
+  /**
+   * Optional hint text.
+   */
+  @Prop() hint?: string;
+
+  /**
+   * An optional message that will be read by screen readers when the select is focused.
+   */
+  @Prop() messageAriaDescribedby?: string;
+
+  /**
+   * The event emitted when the selected value changes
+   */
+  @Event() vaSelect: EventEmitter;
+
+  /**
+   * Whether an error message should be shown - set to false when this component is used inside va-date or va-memorable-date
+   * in which the error for the va-select will be rendered outside of va-select
+   */
+  @Prop() showError?: boolean = true;
+
+  /**
+   * The event used to track usage of the component. This is emitted when an
+   * option is selected and enableAnalytics is true.
+   */
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
```

Comment:
I've been advised previously by @jestutt from platform analytics that we should not add analytics code to a component until we have coordinated with her. So I think we should remove any custom analytics code for now and we'll create a follow-up ticket to get it added.

Unless @oleksii-morgun your team needs it for your user testing?

Reply:
no, not for GA analytics. Our team can still track this with DD RUM

Reply:
@oleksii-morgun Sounds good. I think then that you can go ahead and remove the analytics event and other related code and we will create a follow-up ticket to coordinate adding it at a later time.